docs/service-hooks/create-subscription.md

```diff
@@ -15,7 +15,7 @@ ms.date: 08/04/2016
 
 # Create a service hooks subscription programmatically
 
-Using the [Subscriptions](https://visualstudio.microsoft.com/en-us/docs/integrate/api/hooks/subscriptions) REST APIs, you can programmatically create a subscription that performs an action on an external (consumer) service when a specific event occurs in a project. For example, you can create a subscription to notify your service when a build fails.
+Using the [Subscriptions](https://docs.microsoft.com/en-us/rest/api/vsts/hooks/subscriptions) REST APIs, you can programmatically create a subscription that performs an action on an external (consumer) service when a specific event occurs in a project. For example, you can create a subscription to notify your service when a build fails.
```

Comment:
You can even drop the `https://docs.microsoft.com/en-us` since they're site root links

Reply:
Thank you for the suggestion! I have updated the hyperlinks with relative links that reference the corresponding markdown files.